superset-frontend/src/pages/Home/index.tsx

```diff
@@ -352,7 +353,7 @@ function Welcome({ user, addDangerToast }: WelcomeProps) {
 
   return (
     <>
-      <SubMenu {...menuData} />
+      {SubmenuExtension ? <SubmenuExtension /> : <SubMenu {...menuData} />}
```

Comment:
Should the submenu receive the `menuData` props too?

Reply:
Good point, done